benches/bin/utils.rs

```diff
@@ -0,0 +1,77 @@
+extern crate rand;
+
+use std::sync::atomic::{ATOMIC_U64_INIT, AtomicU64, Ordering};
+use std::time::Duration;
+use tikv::coprocessor::codec::table::{encode_index_seek_key, encode_row_key};
+use tikv::util::codec::number::NumberEncoder;
+
+use rand::Rng;
+
+#[inline]
+pub fn next_ts() -> u64 {
+    static CURRENT: AtomicU64 = ATOMIC_U64_INIT;
+    CURRENT.fetch_add(1, Ordering::SeqCst)
+}
+
+/// Generate `count` row keys that all with the specified `table_id`.
+///
+/// Row id will start with `start_id` and increment
+pub fn generate_row_keys(table_id: i64, start_id: i64, count: usize) -> Vec<Vec<u8>> {
+    let mut result = Vec::with_capacity(count);
+    for i in (start_id)..(start_id + count as i64) {
+        let mut handle = Vec::with_capacity(8);
+        handle.encode_i64(i as i64).unwrap();
+        let key = encode_row_key(table_id, &handle);
+        result.push(key);
+    }
+    result
+}
+
+
+/// Generate `count` unique index keys that all with the specified `table_id`, `index_id`,
+/// and `value_len`.
+pub fn generate_unique_index_keys(
```

Comment:
do we need to consider non-unique index?

Reply:
Due to that unique and non-unique index differs slightly, I think there will not be any significant difference between their bench results. For now I wrote only this.

Reply:
It’s better to bench, not only make a hypothesis

Reply:
Maybe this can be done later..